.devcontainer/Dockerfile

```diff
@@ -10,4 +10,6 @@ RUN install-tool python 3.12.0
 # renovate: datasource=npm
 RUN install-tool corepack 0.21.0
 
+RUN pip install setuptools
```

Comment:
how does it relate to the pnpm install?

I don't think we need python at all for local build. 🤔

Reply:
no, re2 should download a prebuild binary, so verify that python is really the cause.

you can simply test it locally by downgrade python.

I'm pretty sure it's something inside pnpm.